master/buildbot/changes/github.py

```diff
@@ -101,21 +101,17 @@ def reconfigService(self,
                         magic_link=False,
                         repository_type="https",
                         github_property_whitelist=None,
+                        debug=False,
+                        verify=False,
                         **kwargs):
         yield base.ReconfigurablePollingChangeSource.reconfigService(
             self, name=self.name, **kwargs)
 
-        if baseURL is None:
-            baseURL = HOSTED_BASE_URL
-        if baseURL.endswith('/'):
-            baseURL = baseURL[:-1]
-
-        http_headers = {'User-Agent': 'Buildbot'}
-        if token is not None:
-            http_headers.update({'Authorization': 'token ' + token})
+        self.debug = debug
+        self.verify = verify
 
-        self._http = yield httpclientservice.HTTPClientService.getService(
-            self.master, baseURL, headers=http_headers)
+        if baseURL is not None and baseURL.endswith('/'):
```

Comment:
this should be handled automatically by GithubAPIService (as this is a code that we would have to use for all its users)